core/src/main/scala/tech/beshu/ror/accesscontrol/factory/RawRorConfigBasedCoreFactory.scala

```diff
@@ -180,7 +180,7 @@ class RawRorConfigBasedCoreFactory(implicit clock: Clock,
     rulesNelDecoder(definitions)
       .toSyncDecoder
       .emapE { rules =>
-        RulesValidator.validate(rules) match {
+        RulesValidator.validate(rules.sorted) match {
```

Comment:
I wonder if we shouldn't change RulesValidator to BlocksValidator and pass there whole block. At the moment we have sorting of rules in two places. This is fragile part of the ROR code, so maybe we should be careful here. And it seems that we rather validate block, not rules per se.

Reply:
I created factory method where I'm validating passed rules. I think we don't want to create block and then proceed with validation, but instead validate all data before creation, so we can not create block in invalid state, right? :) I didn't make constructor for block private because it's used in tests...but maybe I should and then use factory method there as well?